javascript/javascript1/week1/Homework/smartEase.js

```diff
@@ -0,0 +1,145 @@
+//********************************************************** */
+/* Step 2: Javascript warm up part one
+              --FreeCodeCamp--
+      https://www.freecodecamp.org/BogdyAlex87*/
+
+//*************************************************************/
+
+/*Step 3: Smart-ease
+          Age-ify (A future age calculator)*/
+//******************************************************************
+
+const yearOfBirth = 1987;
+const yearFuture = 2059;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+/*Goodboy-Oldboy (A dog age calculator)*/
+
+const dogYearOfBirth = 2020;
+const dogYearFuture = 2024;
+const humanAge = dogYearFuture - dogYearOfBirth;
+
+let dogAge;
+
+if (humanAge === 1) {
+  dogAge = 15;
+} else if (humanAge === 2) {
+  dogAge = 24;
+} else {
+  dogAge = 24 + (humanAge - 2) * 5;
+}
+
+const shouldShowResultInDogYears = true;
+
+const dogYear = shouldShowResultInDogYears ? dogAge : humanAge;
+console.log(
+  "Your dog will be ",
+  dogYear,
+  shouldShowResultInDogYears
+    ? " dog years old in " + dogYearFuture
+    : "human year old in " + dogYearFuture
+);
+
+/*Housey pricey (A house price estimator)*/
+
+// Peter's house details
+const peterWidth = 8;
```

Comment:
Those variable names can be confusing unless someone is familiar with the assignment. `peterHouseWidth` might be more precise.

Reply:
Changed.